Dizzy/Screens/Discovery/DiscoveryVC.swift

```diff
@@ -14,14 +14,18 @@ class DiscoveryVC: ViewController {
     let topBar = DiscoveryTopBar()
     let themeImageView = UIImageView()
     let nearByPlacesView = NearByPlacesView()
-    let viewModel: DiscoveryViewModelType
+    var viewModel: DiscoveryViewModelType
+    
+    let nearByPlacesViewCornerRadius = CGFloat(5)
     
     init(viewModel: DiscoveryViewModelType) {
         self.viewModel = viewModel
         super.init()
         addSubviews()
         layoutViews()
         setupViews()
+        bindViewModel()
+        self.viewModel.delegate = self
```

Comment:
ViewControllers should not be delegates of the viewModels, I think it will be better if you will use observable, and when the data is updated then you reloadData.

Reply:
ok